blockchain/tx_pool.go

```diff
@@ -739,6 +739,11 @@ func (pool *TxPool) validateTx(tx *types.Transaction) error {
 				return ErrInsufficientFundsFeePayer
 			}
 		}
+		// balance check in case of sender = feepayer
```

Comment:
It would be helpful if you add a reason why this additional sender balance check is needed. What about this?
```suggestion
		// additional sender balance check in case of sender = feepayer
		// since a single account has to bear the both cost(feepayer_cost + sender_cost),
		// it is necessary to check whether the balance is equal to the sum of the cost.
```

Reply:
Thx!